### PR TITLE
CTAA-1307: storyboard link

### PR DESCRIPTION
**Pull request by bastianh**

### CODE REVIEW

File: CoronaContact/Resources/Localisation/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -230,6 +230,7 @@
 "sickness_certificate_attest_headline"                                       = "Infection reported";
 "sickness_certificate_attest_description"                                    = "Please remain in quarantine. You have officially declared yourself ill with the coronavirus.";
 "sickness_certificate_attest_button"                                         = "Self-quarantine";
+"sickness_certificate_attest_button_revoke"                                  = "Krankmeldung zurückziehen";
```
Comment (finnsch) [inline CoronaContact/Resources/Localisation/en.lproj/Localizable.strings:233]: Is the English translation not ready yet?

File: CoronaContact/Scenes/Main/MainCoordinator.swift [hunk 2]
```diff
@@ -89,6 +89,12 @@ class MainCoordinator: Coordinator, ShareSheetPresentable {
         child.start()
     }
 
+    func revokateSickness() {
```
Comment (finnsch) [inline CoronaContact/Scenes/Main/MainCoordinator.swift:92]: The correct naming would be `revoke` instead of `revokate`, right?

Comment (bastianh) [reply CoronaContact/Scenes/Main/MainCoordinator.swift:92]: yes :)

File: CoronaContact/Resources/Localisation/en.lproj/Localizable.strings (hunk 1 above)
Comment (bastianh) [reply CoronaContact/Resources/Localisation/en.lproj/Localizable.strings:233]: looks like it is missing in the translation table

**Pull request merged**